go/staking/api/slashing.go

```diff
@@ -11,20 +13,37 @@ type SlashReason int
 const (
 	// SlashDoubleSigning is slashing due to double signing.
 	SlashDoubleSigning SlashReason = 0
+	SlashMax                       = SlashDoubleSigning
```

Comment:
I don't think we use this anywhere so it could just be removed.

Reply:
yeah indeed it's not :+1: